connect-examples/v2/node_bookings/public/stylesheets/style.css

```diff
@@ -424,6 +444,10 @@ a:visited {
   cursor: pointer;
 }
 
+.sq-form-control .button.btn-primary:hover {
+  background-color: #005FE6;
```

Comment:
This is super nit-picky so feel free not to change it, but some CSS style guides recommend using all [lower-case for color hex values](https://google.github.io/styleguide/htmlcssguide.html#Capitalization). Personally I don't think it matters all the much, but whatever we have should be internally consistent.